tests/trinity/core/config/test_trinity_config_object.py

```diff
@@ -1,92 +0,0 @@
-from pathlib import Path
```

Comment:
Extracting this into a small PR that only moves this test file into the new location under `tests/trinity/core/configuration/test_trinity_config_object.py`  will help reduce the overall size of this diff.